packages/@react-aria/actiongroup/src/ActionGroupKeyboardDelegate.ts

```diff
@@ -17,11 +17,13 @@ export class ActionGroupKeyboardDelegate<T> implements KeyboardDelegate {
   private collection: Collection<T>;
   private flipDirection: boolean;
   private disabledKeys: Set<Key>;
+  private isDisabled: boolean;
 
   constructor(collection: Collection<T>, direction: Direction, orientation: Orientation, disabledKeys: Set<Key> = new Set()) {
     this.collection = collection;
     this.flipDirection = direction === 'rtl' && orientation === 'horizontal';
     this.disabledKeys = disabledKeys;
+    this.isDisabled = collection.size === disabledKeys.size;
```

Comment:
We should probably handle the case where `disabledKeys` includes keys that don't exist in the collection for some reason. Just checking the length won't tell us this.